sycl/include/sycl/types.hpp

```diff
@@ -272,30 +272,6 @@ template <typename T> using vec_data = detail::vec_helper<T>;
 template <typename T>
 using vec_data_t = typename detail::vec_helper<T>::RetType;
 
-#if !defined(__INTEL_PREVIEW_BREAKING_CHANGES)
-
-#if defined(_WIN32) && (_MSC_VER)
-// MSVC Compiler doesn't allow using of function arguments with alignment
-// requirements. MSVC Compiler Error C2719: 'parameter': formal parameter with
-// __declspec(align('#')) won't be aligned. The align __declspec modifier
-// is not permitted on function parameters. Function parameter alignment
-// is controlled by the calling convention used.
-// For more information, see Calling Conventions
-// (https://docs.microsoft.com/en-us/cpp/cpp/calling-conventions).
-// For information on calling conventions for x64 processors, see
-// Calling Convention
-// (https://docs.microsoft.com/en-us/cpp/build/x64-calling-convention).
-#pragma message("Alignment of class vec is not in accordance with SYCL \
-specification requirements, a limitation of the MSVC compiler(Error C2719).\
-Requested alignment applied, limited at 64.")
-#define __SYCL_ALIGNED_VAR(type, x, var)                                       \
-  type __declspec(align((x < 64) ? x : 64)) var
-#else
-#define __SYCL_ALIGNED_VAR(type, x, var) alignas(x) type var
```

Comment:
Ok, I see now why we need that macro :) Please expect another attempt at unifying this in the next commit